src/node.cc

```diff
@@ -1423,7 +1423,8 @@ bool IsExceptionDecorated(Environment* env, Local<Value> er) {
 
 void AppendExceptionLine(Environment* env,
                          Local<Value> er,
-                         Local<Message> message) {
+                         Local<Message> message,
+                         enum ErrorHandlingMode mode) {
```

Comment:
The `enum` keyword is not strictly necessary (but it doesn't hurt and I realize we use it in other places.)